setup.py

```diff
@@ -21,6 +22,11 @@
 print('found these packages:', pkgs)
 
 schema_dir = 'data'
+
+# HACK: numpy needs to be installed before scipy
+pip.main(['install', 'numpy'])
+pip.main(['install', 'scipy'])
```

Comment:
This should be done extending the Install command and not directly here. 

https://cbuelter.wordpress.com/2015/10/25/extend-the-setuptools-install-command/
https://github.com/scikit-build/scikit-build/tree/master/skbuild/command